aim/db/migration/alembic_migrations/versions/bca1ef645fe1_epg_sync_flag.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) 2018 Cisco Systems
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+"""Add sync flag to EndpointGroup
+
+Revision ID: bca1ef645fe1
+Revises: acebbacca3666
+Create Date: 2018-01-10 12:23:39.608507
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'bca1ef645fe1'
+down_revision = '1e3fda0945f2'
+branch_labels = None
+depends_on = None
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+
+    op.add_column(
+        'aim_endpoint_groups',
+        sa.Column('sync', sa.Boolean, server_default=sa.literal(True))
+    )
+    op.add_column(
```

Comment:
since we are adding this sequence_number also in this migration script, we should probably also enhance the file naming to reflect this change. Also I don't see the HEAD file being changed though.....